tests/arguments.rs

```diff
@@ -56,7 +56,7 @@ fn argument_with_default() {
 fn argument_with_raw_default() {
     #[derive(StructOpt, PartialEq, Debug)]
     struct Opt {
-        #[structopt(raw(default_value = r#""42""#))]
+        #[structopt(default_value = "42")]
```

Comment:
The test is now quite useless as the indent was to test raw default.

Reply:
Indeed. Removed.